build.gradle

```diff
@@ -47,7 +47,10 @@ plugins {
 
 ext {
   gradleVersion = versions.gradle
-  minJavaVersion = 11
+  minClientJavaVersion = 11
+  minServerJavaVersion = 17
```

Comment:
I think it would be more accurate to say `minNonClientJavaVersion` as this includes things like `tools` and other such modules.

Reply:
Make sense 😄

Reply:
minNonClientJavaVersion looks good to me. Additionally, the connect:api module belongs to client APIs, but in KIP-1032 we decided to bump the Connect module to JDK 17. Could you please add a comment for the Connect module?